version

```diff
@@ -1 +1 @@
-3.2.11
+3.2.12
```

Comment:
Please don't increment version, especially not in the same commit as other changes.